internal/indexer/models/page.py

```
@@ -0,0 +1,42 @@
+@dataclass
+class Page:
+    normalized_url: str
+    html: str
+    content_type: str
+    status_code: int
+    last_crawled: datetime
+
+    @classmethod
+    def from_hash(cls, page_data: Dict[str, Any]) -> 'Page':
+
+        if page_data == None:
```

Ruff (0.15.9)

[error] Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)